src/lib.rs

```diff
@@ -526,7 +533,47 @@ mod test {
         .unwrap();
     }
 
-    #[rstest(hist_batches, case::incremental(&[1, 2]), case::replay(&[2]))]
+    #[rstest(core,
+    case::incremental(single_activity_setup(& [1, 2])),
+    case::replay(single_activity_setup(& [2]))
```

Comment:
The weirdo formatting here with the space after `&` and lack of indentation would be nice to fix

Reply:
Fixed.